### PR TITLE
[OPSRC-536] Add Image property to ShippingMethod

### PR DESCRIPTION
**Pull request by DominikWolos**

### CODE REVIEW

File: src/EventListener/ImageUploadListener.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        $this->uploader = $uploader;
+    }
+
+    public function uploadImages(GenericEvent $event): void
+    {
+        $subject = $event->getSubject();
+
+        Assert::isInstanceOf($subject, ImageAwareInterface::class);
+
+        if (empty($subject->getImage()) || !$subject->getImage()->hasFile()) {
```
Comment (senghe) [inline src/EventListener/ImageUploadListener.php:35]: We should avoid using `empty(...)` function :)

File: src/Resources/views/Admin/ShippingMethod/_image.html.twig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{% if shipping_method.image is not empty and shipping_method.image.path is not empty %}
```
Comment (senghe) [inline src/Resources/views/Admin/ShippingMethod/_image.html.twig:1]: Does 'empty' in twig realize the 'empty(...)' php function? If so, I would check it different way

File: tests/Behat/Context/Ui/Admin/ShippingMethodContext.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusInPostPlugin\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Tests\BitBag\SyliusInPostPlugin\Behat\Page\Admin\ShippingMethod\UpdatePageInterface;
+
+class ShippingMethodContext implements Context
+{
+    private UpdatePageInterface $updatePage;
```
Comment (senghe) [inline tests/Behat/Context/Ui/Admin/ShippingMethodContext.php:18]: No empty line

**Review by senghe — CHANGES_REQUESTED** [review]

File: src/EventListener/ImageUploadListener.php (hunk 1 above)
Comment (DominikWolos) [reply src/EventListener/ImageUploadListener.php]: Replaced with null comparision

File: src/Resources/views/Admin/ShippingMethod/_image.html.twig (hunk 2 above)
Comment (DominikWolos) [reply src/Resources/views/Admin/ShippingMethod/_image.html.twig]: Replaced with null comparision

**Review by senghe — APPROVED** [review]

**Pull request merged**